etebase_server/settings.py

```diff
@@ -162,6 +162,8 @@
 
     if "allowed_hosts" in config:
         ALLOWED_HOSTS = [y for x, y in config.items("allowed_hosts")]
+        CSRF_TRUSTED_ORIGINS = ["https://" + y for x, y in config.items("allowed_hosts")] + \
```

Comment:
Allowed hosts is a list of hosts, it shouldn't have http/https, and it should work across both.

Reply:
it is called an "origin", and the docs say that http(s):// needs to be added:

> ..If you need cross-origin unsafe requests, continuing the example, add 'https://subdomain.example.com' to this list (and/or http://... if requests originate from an insecure page).

https://docs.djangoproject.com/en/4.2/ref/settings/#csrf-trusted-origins

Reply:
Yeah, I stand corrected, it's a breaking change in Django 4.0, and since we now support django 4.0 it should be fixed. :)